Sources/xcodeproj/Objects/Project/PBXProject.swift

```diff
@@ -191,6 +191,44 @@ public final class PBXProject: PBXObject {
 
         return reference
     }
+    
+    
+    /// Adds a local swift package
+    ///
+    /// - Parameter path: Relative path to the swift package
+    /// - Parameter productName: The product to depend on
```

Comment:
Does it require the extension to be in the name/

Reply:
Not sure I follow, could you elaborate?

Reply:
If the product name is for example `Alamofire.framework`, do I have to pass `Alamofire` or the name with the extension? I'd include that in the line that documents the argument:
```swift
/// - Parameter productName: The product to depend on with/without the extension
```

Reply:
Without the extension, added it 👍